src/store/modules/nutrition/getters.ts

```diff
@@ -1,7 +1,45 @@
 import { GetterTree } from 'vuex';
 
-export const getters: GetterTree<any, any> = {
-    cachedDailyIntake: (state: any) => (day: string) => {
+import { RootState, NutritionState } from '@/models/store';
+import { FoodInfo, FoodNutrient, Meal } from '@/models/nutrition';
+
+export const getters: GetterTree<NutritionState, RootState> = {
+    cachedDailyIntake: (state: NutritionState) => (day: string) => {
         return state.intakes[day];
     },
+    foodInfo: (state: NutritionState) => (id: string): FoodInfo | undefined => {
+        return state.foodInfoArray.length > 0
+            ? state.foodInfoArray.find((foodInfo: FoodInfo) => foodInfo.resource_id === id)
+            : undefined;
+    },
+    getMealFromCurrentDay: (state: NutritionState) => (mealName: string): Meal | null => {
+        const mealIndex = state.currentIntake.meals && state.currentIntake.meals
```

Comment:
I wouldn't use that syntax. I know how it works but it's not really obvious at first and can be misleading. I prefer clear syntax even if it takes few more lines.

Reply:
I understand, will change

Reply:
Done